bigbluebutton-html5/imports/ui/components/actions-bar/create-breakout-room/container.jsx

```diff
@@ -5,12 +5,24 @@ import BreakoutRoomService from '/imports/ui/components/breakout-room/service';
 
 import CreateBreakoutRoomModal from './component';
 
+const METEOR_SETTINGS_APP = Meteor.settings.public.app;
+
 const CreateBreakoutRoomContainer = (props) => {
+  const startWithAllowUserToChooseABreakout = METEOR_SETTINGS_APP.breakouts.startWithAllowUserToChooseABreakout;
```

Comment:
We can ensure the corresponding capture is enabled here:
```javascript
  const { setAllowUserToChooseABreakout } = METEOR_SETTINGS_APP.breakouts;
  const setCaptureBreakoutWhiteboard = METEOR_SETTINGS_APP.breakouts.setCaptureBreakoutWhiteboard
                                    && isImportPresentationWithAnnotationsFromBreakoutRoomsEnabled;
  const setCaptureBreakoutNotes = METEOR_SETTINGS_APP.breakouts.setCaptureBreakoutWhiteboard
                                    && isImportSharedNotesFromBreakoutRoomsEnabled;
```

Note that I changed the config names to match what is used throughout the backend.

Reply:
Great! I already accepted your commit. I'm just going to test and then request your review again.

Thank you for the suggestions and feedback, anything else do not hesitate to comment!